Marlin/Configuration_adv.h

```diff
@@ -3399,58 +3397,66 @@
     #define LASER_TEST_PULSE_MIN           1   // Used with Laser Control Menu
     #define LASER_TEST_PULSE_MAX         999   // Caution: Menu may not show more than 3 characters
 
-    /**
-     * Enable inline laser power to be handled in the planner / stepper routines.
-     * Inline power is specified by the I (inline) flag in an M3 command (e.g., M3 S20 I)
-     * or by the 'S' parameter in G0/G1/G2/G3 moves (see LASER_MOVE_POWER).
-     *
-     * This allows the laser to keep in perfect sync with the planner and removes
-     * the powerup/down delay since lasers require negligible time.
-     */
-    //#define LASER_POWER_INLINE
-
-    #if ENABLED(LASER_POWER_INLINE)
+    #define SPINDLE_LASER_POWERUP_DELAY     50 // (ms) Delay to allow the spindle/laser to come up to speed/power
+    #define SPINDLE_LASER_POWERDOWN_DELAY   50 // (ms) Delay to allow the spindle to stop
+  
+  
+    /**#define LASER_POWER_INLINE <- This define is no longer required
+    * Any M3 G1/2/3/5 gcode that includes an I parameter will enable continuous inline power mode
+    * e.g M3 I enables continuous inline power and it is processed by the planner. Move blocks are 
+    * calculated and each block power buffer value is set there. The Stepper ISR then processes the blocks inline.
+    * M4I sets dynamic mode which taked the currently set feedrate and calculated a laser power OCR value.
+    * Setting a powered move G1/2/3/5 gcode F parameter sets a porportional power output from 1-255, S1-65535 respectively
```

Comment:
The `F` parameter specifies the feedrate, so this description needs to be clarified to explain how the feedrate set by `F` translates into a "proportional power output" and what is meant by "S1-65535 respectively." This unclear description implies that `F` does not set the feedrate but does "something else."

Reply:
Updated